### PR TITLE
EPR: I'll add this feature later lmao

### PR DESCRIPTION
**Pull request by LucunJi**
Now causing mixin conflicts after a major change in EPR 3.0.0

### CODE REVIEW

File: src/main/java/me/fallenbreath/tweakermore/mixins/tweaks/mod_tweaks/eprHideOnDebugHud/PlayerHUDRenderer_v2Mixin.java [hunk 1]
```diff
@@ -31,12 +31,12 @@
...
 
 @SuppressWarnings("UnresolvedMixinReference")
-@Restriction(require = @Condition(value = ModIds.extra_player_renderer, versionPredicates = ">=2.0.0"))
+@Restriction(require = @Condition(value = ModIds.extra_player_renderer, versionPredicates = "[2.0.0, 2.4.1]"))
 @Pseudo
 @Mixin(targets = "github.io.lucunji.explayerenderer.client.render.PlayerHUDRenderer")
 public abstract class PlayerHUDRenderer_v2Mixin
 {
-	@Inject(method = "doRender", at = @At("HEAD"), remap = false, cancellable = true)
+	@Inject(method = "doRender", at = @At("HEAD"), remap = false, cancellable = true, require = 0)
```
Comment (Fallen-Breath) [inline src/main/java/me/fallenbreath/tweakermore/mixins/tweaks/mod_tweaks/eprHideOnDebugHud/PlayerHUDRenderer_v2Mixin.java:39]: I think `require = 0` doesn't help here cuz it's targeting `HEAD`

**Comment by Fallen-Breath** [comment]
It can also be a `PlayerHUDRenderer_v3Mixin` XD

**Comment by LucunJi** [comment]
jitpack is giving me timeout for `com.github.Fallen-Breath:lazydfu:a7cfc44c0c`, can't really build the mod

**Comment by LucunJi** [comment]
experimented with `lazydfu` in local maven, `[*,*]` or `[*,*)` doesn't work, and the best range is `^2.0.0` (`<3.0.0` fails to ignore alpha versions)

`require = 0` indeed works

**Review by Fallen-Breath — APPROVED** [review]
LGTM

**Pull request merged**